files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -329,6 +329,15 @@
     regexp: '^(session[\s]+optional[\s]+pam_motd\.so.*)$'
     replace: '#\1'
 
+#
+# Restricting su access to ensure only authorized users in a specific group
+# can elevate privileges to another user, like the root user
+#
+- replace:
+    dest: /etc/pam.d/su
+    regexp: '^#?[\s]*(auth[\s]+required[\s]+pam_wheel\.so.*)$'
```

Comment:
After this change, does `sudo su` still work without a password from the `delphix` user?

Reply:
Yes, with this change `sudo su` works without password from `delphix `user.